src/accounts/accountModel.java

```diff
@@ -0,0 +1,121 @@
+package accounts;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author aswecker
+ */
+public class accountModel implements Initializable {
+
+    private static final int ID = 1;
+    private static final int First_Name = 1;
+    private static final int Last_Name = 1;
+    private static final int Username = 1;
+    private static final int Password = 1;
+    
+    private static final String COMMA_DELIMITER = ",";
+    private static final String NEW_LINE = "\n";
+    private static final String FILE_HEADER = "Id,First_Name,Last_Name,Username,Password";
+    
+  
+
+   
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        addUser();
+        readUser();
+    }
+
+    // Add a user
+    public void addUser() {
+        User u = new User(1, "Test", "User", "TestUser", "password");
+        User u1 = new User(2, "John", "Doe", "JohnDoe", "password");
+        List<User> Users = new ArrayList<User>();
+
+        Users.add(u);
+        Users.add(u1);
+
+        File file = new File("data/Accounts.csv");
+        FileWriter writer = null;
+
+        try {
+            writer = new FileWriter(file);
```

Comment:
This will cause the existing file to be overwritten, erasing existing accounts. The easiest thing to do is to append the file instead: https://docs.oracle.com/javase/7/docs/api/java/io/FileWriter.html#FileWriter(java.io.File,%20boolean)